src/fmu/dataio/providers/objectdata/_base.py

```diff
@@ -226,7 +226,7 @@ def _get_named_stratigraphy(self) -> NamedStratigraphy:
             top=named_stratigraphy.get("top"),
             base=named_stratigraphy.get("base"),
         )
-        if rv.name != "name":
+        if name not in rv.alias:
```

Comment:
Not necessary to handle the `!= "name"` after the recent refactoring of the code. It will now only enter this part if name is present in the `stratigraphy `